### DISCUSSION TITLE
Dynamic Interval

### DISCUSSION THREAD

**Discussion by C0NTi** [mKeRix/room-assistant#475] [Ideas]
It would be great if I could adjust how aggressive room assistant is on the fly.

For example It would be good if I can have all my instances polling every 6 seconds when my house is marked as empty - So that on my return I get a much more responsive detection when I get home. 

However when I am home It would be nice if I could then change the instances to poll every 20 seconds say and then overnight trigger it to scan eve more infrequently.

I know you have introduced the inquiry switch, however this reduces my presence detection to on/off, I would like to have a bit more control over it and choose from good/slow presence detections.

**Comment by mKeRix** [comment]
I think that is a cool idea and something I would like to see become reality sometime! Could you open a feature request ticket for this? You can just link to this discussion if you want, so that you don't need to type everything again. It just makes it easier to track the todo list. :)
